src/wp-includes/class-wp-query.php

```diff
@@ -3193,12 +3193,17 @@ public function get_posts() {
 					} elseif ( 'id=>parent' === $q['fields'] ) {
 						_prime_post_parent_id_caches( $post_ids );
 
+						$post_parent_cache_keys = array();
+						foreach ( $post_ids as $post_id ) {
+							$post_parent_cache_keys[] = 'post_parent::' . (string) $post_id;
```

Comment:
Why are we using `::` here. Other similar seprators using a single `:`

https://github.com/WordPress/wordpress-develop/blob/d95c0e1c79739b3360c50821ee215cda343e0cbc/src/wp-includes/class-wp-query.php#L4878

https://github.com/WordPress/wordpress-develop/blob/d95c0e1c79739b3360c50821ee215cda343e0cbc/src/wp-includes/class-wp-site-query.php#L360

I would recommend following other cache key generations.

Reply:
> Why are we using `::` here. Other similar seprators using a single `:`

I've changed this, it's totally me having a dumb moment.